Logger/FileLogger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    class FileLogger : BaseLogger
+    {
+        private string filepath;
```

Comment:
Fields should be named underscore PascalCase
```suggestion
        private string _FilePath;
```